boards/atxmega-a1u-xpro/Makefile.include

```diff
@@ -1,4 +1,7 @@
 AVRDUDE_PROGRAMMER = xplainedpro_pdi
 DEBUGPROTO = -X
 
+RAM_LEN = 64K
```

Comment:
This needs at least a comment.
It's not  immediately obvious how 8k SRAM + 512k external RAM = 64k RAM_LEN

Ideally you could have `EXT_RAM_LEN = 512k` here and do the calculation of the true usable RAM size in `cpu/avr8_common` or `cpu/atxmega`

Reply:
Added docs and fixed a bug. The memory can be only expanded when 1 cs[x].address is 0.